avocado/utils/software_manager.py

```diff
@@ -301,7 +301,7 @@ def rpm_install(self, file_path):
         try:
             process.system(cmd)
             return True
-        except process.CmdError, details:
+        except process.CmdError as details:
```

Comment:
This seems to be the same changes as https://github.com/avocado-framework/avocado/pull/2383/commits/97ac74ca72a1cc6876b083174af6a0bcb7538326

Reply:
Which are now here: https://github.com/avocado-framework/avocado/pull/2388/commits/8bd578ddaea5e9ea528f45077ec73c28e523a020

https://github.com/avocado-framework/avocado/pull/2388

Reply:
OK, let me force-drop it...